### ISSUE TITLE
Angular 11 - types in html files are not shown

### ISSUE THREAD

**Issue by jtsom** [angular/vscode-ng-language-service#1002]
When editing an HTML file, the types of variables are not known, or shown. Hovering over a property of a component, just shows as "typescript".

**To Reproduce**

1. Create or Open a new Angular 11 application.
2. Open the app.component.html file and hover over the {{ title }} member variable.

The type shown is just "typescript"

**Expected behavior**
The type should show the proper Typescript type - "string' in this case.

**Logs**

Please attach two logs:

1. Console output (Go to toolbar --> View --> Output, and copy the contents)
```
[Info  - 1:41:20 PM] Angular language server process ID: 18500
[Info  - 1:41:20 PM] Using typescript/lib/tsserverlibrary v4.0.5 from c:\Users\John Tsombakos\.vscode-insiders\extensions\angular.ng-template-0.1100.1\node_modules\typescript\lib\tsserverlibrary.js
[Info  - 1:41:20 PM] Using @angular/language-service v11.0.1 from c:\Users\John Tsombakos\.vscode-insiders\extensions\angular.ng-template-0.1100.1\server\node_modules\@angular\language-service\bundles\language-service.js
[Info  - 1:41:20 PM] Log file: c:\Users\John Tsombakos\AppData\Roaming\Code - Insiders\logs\20201201T134113\exthost1\Angular.ng-template\nglangsvc.log
[Info  - 1:41:33 PM] Enabling VE language service for c:/Users/John Tsombakos/source/ng11-test/tsconfig.json.

```
2. Log file (Path is printed to the console output at startup)
```
Info 0    [13:41:20.46] Format host information updated
Info 1    [13:41:20.75] reload projects.
Info 2    [13:41:20.75] Before ensureProjectForOpenFiles:
Info 3    [13:41:20.75] After ensureProjectForOpenFiles:
Info 4    [13:41:20.75] Host file extension mappings updated
Info 5    [13:41:20.77] Angular language server process ID: 18500
Info 6    [13:41:20.77] Using typescript/lib/tsserverlibrary v4.0.5 from c:\Users\John Tsombakos\.vscode-insiders\extensions\angular.ng-template-0.1100.1\node_modules\typescript\lib\tsserverlibrary.js
Info 7    [13:41:20.77] Using @angular/language-service v11.0.1 from c:\Users\John Tsombakos\.vscode-insiders\extensions\angular.ng-template-0.1100.1\server\node_modules\@angular\language-service\bundles\language-service.js
Info 8    [13:41:20.78] Log file: c:\Users\John Tsombakos\AppData\Roaming\Code - Insiders\logs\20201201T134113\exthost1\Angular.ng-template\nglangsvc.log
Info 9    [13:41:20.213] Search path: c:/Users/John Tsombakos/source/ng11-test/src/app
Info 10   [13:41:20.214] For info: c:/Users/John Tsombakos/source/ng11-test/src/app/app.component.html :: Config file name: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 11   [13:41:20.215] Opened configuration file c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 12   [13:41:20.219] Loading new project: Creating possible configured project for c:/Users/John Tsombakos/source/ng11-test/src/app/app.component.html to open
Info 13   [13:41:20.256] Config: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json : {
 "rootNames": [
  "c:/Users/John Tsombakos/source/ng11-test/e2e/src/app.e2e-spec.ts",
  "c:/Users/John Tsombakos/source/ng11-test/e2e/src/app.po.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/main.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/polyfills.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/test.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/app/app-routing.module.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/app/app.component.spec.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/app/app.component.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/app/app.module.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/app/data.service.spec.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/app/data.service.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/app/types.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/environments/environment.prod.ts",
  "c:/Users/John Tsombakos/source/ng11-test/src/environments/environment.ts"
 ],
 "options": {
  "baseUrl": "c:/Users/John Tsombakos/source/ng11-test",
  "outDir": "c:/Users/John Tsombakos/source/ng11-test/dist/out-tsc",
  "forceConsistentCasingInFileNames": true,
  "strict": true,
  "noImplicitReturns": true,
  "noFallthroughCasesInSwitch": true,
  "sourceMap": true,
  "declaration": false,
  "downlevelIteration": true,
  "experimentalDecorators": true,
  "moduleResolution": 2,
  "importHelpers": true,
  "target": 2,
  "module": 6,
  "lib": [
   "lib.es2018.d.ts",
   "lib.dom.d.ts"
  ],
  "configFilePath": "c:/Users/John Tsombakos/source/ng11-test/tsconfig.json"
 }
}
Info 14   [13:41:20.258] Loading global plugin @angular/language-service
Info 15   [13:41:20.258] Enabling plugin @angular/language-service from candidate paths: c:\Users\John Tsombakos\.vscode-insiders\extensions\angular.ng-template-0.1100.1\server\node_modules\@angular\language-service\bundles\language-service.js,c:/Users/John Tsombakos/.vscode-insiders/extensions/angular.ng-template-0.1100.1/node_modules/typescript/lib/tsserverlibrary.js/../../..
Info 16   [13:41:20.258] Loading @angular/language-service from c:\Users\John Tsombakos\.vscode-insiders\extensions\angular.ng-template-0.1100.1\server\node_modules\@angular\language-service\bundles\language-service.js (resolved to c:/Users/John Tsombakos/.vscode-insiders/extensions/angular.ng-template-0.1100.1/server/node_modules/@angular/language-service/bundles/language-service.js/node_modules)
Info 17   [13:41:20.368] Plugin validation succeded
Info 18   [13:41:20.582] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 19   [13:41:32.314] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 1 structureChanged: true Elapsed: 11732ms
Info 20   [13:41:32.314] Project 'c:/Users/John Tsombakos/source/ng11-test/tsconfig.json' (Configured)
Info 21   [13:41:32.314] 	Files (312)

Info 22   [13:41:32.314] -----------------------------------------------
Info 23   [13:41:32.379] Starting updateGraphWorker: Project: /dev/null/autoImportProviderProject1*
Info 24   [13:41:33.106] Finishing updateGraphWorker: Project: /dev/null/autoImportProviderProject1* Version: 1 structureChanged: true Elapsed: 727ms
Info 25   [13:41:33.106] Project '/dev/null/autoImportProviderProject1*' (AutoImportProvider)
Info 26   [13:41:33.106] 	Files (69)

Info 27   [13:41:33.106] -----------------------------------------------
Info 28   [13:41:33.108] Enabling VE language service for c:/Users/John Tsombakos/source/ng11-test/tsconfig.json.
Info 29   [13:41:33.113] FileWatcher:: Triggered with c:/Users/John Tsombakos/source/ng11-test/src/app/data.service.ts 1:: WatchInfo: c:/Users/John Tsombakos/source/ng11-test/src/app/data.service.ts 500 undefined Project:  WatchType: Closed Script info
Info 30   [13:41:33.114] Elapsed:: 1ms FileWatcher:: Triggered with c:/Users/John Tsombakos/source/ng11-test/src/app/data.service.ts 1:: WatchInfo: c:/Users/John Tsombakos/source/ng11-test/src/app/data.service.ts 500 undefined Project:  WatchType: Closed Script info
Info 31   [13:41:33.121] FileWatcher:: Triggered with c:/Users/John Tsombakos/source/ng11-test/src/app/types.ts 1:: WatchInfo: c:/Users/John Tsombakos/source/ng11-test/src/app/types.ts 500 undefined Project:  WatchType: Closed Script info
Info 32   [13:41:33.121] Elapsed:: 0ms FileWatcher:: Triggered with c:/Users/John Tsombakos/source/ng11-test/src/app/types.ts 1:: WatchInfo: c:/Users/John Tsombakos/source/ng11-test/src/app/types.ts 500 undefined Project:  WatchType: Closed Script info
Info 33   [13:41:33.315] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 34   [13:41:33.321] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 2 structureChanged: false Elapsed: 6ms
Info 35   [13:41:35.632] Before ensureProjectForOpenFiles:
Info 36   [13:41:35.632] After ensureProjectForOpenFiles:
Info 37   [13:46:36.722] Search path: c:/Users/John Tsombakos/source/ng11-test/src/app
Info 38   [13:46:36.722] For info: c:/Users/John Tsombakos/source/ng11-test/src/app/app.component.ts :: Config file name: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 39   [13:46:42.872] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 40   [13:46:42.949] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 3 structureChanged: false Elapsed: 77ms
Info 41   [13:46:42.949] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 42   [13:46:45.260] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 43   [13:46:45.403] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 4 structureChanged: false Elapsed: 143ms
Info 44   [13:46:45.403] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 45   [13:46:45.689] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 46   [13:46:45.797] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 5 structureChanged: false Elapsed: 108ms
Info 47   [13:46:45.797] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 48   [13:46:45.898] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 49   [13:46:45.956] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 6 structureChanged: false Elapsed: 58ms
Info 50   [13:46:45.957] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 51   [13:46:46.226] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 52   [13:46:46.282] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 7 structureChanged: false Elapsed: 55ms
Info 53   [13:46:46.282] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 54   [13:46:46.908] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 55   [13:46:46.952] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 8 structureChanged: false Elapsed: 44ms
Info 56   [13:46:46.952] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 57   [13:46:47.523] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 58   [13:46:47.584] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 9 structureChanged: false Elapsed: 61ms
Info 59   [13:46:47.584] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 60   [13:46:48.60] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 61   [13:46:48.123] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 10 structureChanged: false Elapsed: 63ms
Info 62   [13:46:48.123] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 63   [13:46:51.488] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 64   [13:46:51.599] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 11 structureChanged: false Elapsed: 111ms
Info 65   [13:46:51.599] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 66   [13:46:52.143] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 67   [13:46:52.189] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 12 structureChanged: false Elapsed: 46ms
Info 68   [13:46:52.189] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 69   [13:46:53.172] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 70   [13:46:53.219] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 13 structureChanged: false Elapsed: 47ms
Info 71   [13:46:53.219] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 72   [13:46:54.558] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 73   [13:46:54.602] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 14 structureChanged: false Elapsed: 44ms
Info 74   [13:46:54.602] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 75   [13:46:55.128] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 76   [13:46:55.188] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 15 structureChanged: false Elapsed: 60ms
Info 77   [13:46:55.188] Different program with same set of files:: oldProgram.structureIsReused:: 2
Info 78   [13:47:1.390] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 79   [13:47:1.394] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 16 structureChanged: false Elapsed: 4ms
Info 80   [13:47:2.25] Starting updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json
Info 81   [13:47:2.28] Finishing updateGraphWorker: Project: c:/Users/John Tsombakos/source/ng11-test/tsconfig.json Version: 17 structureChanged: false Elapsed: 3ms

```
**Screenshots**

![image](https://user-images.githubusercontent.com/13488/100783880-5b267200-33dc-11eb-9191-1f4f67f616bb.png)

**Additional context**

**Comment by xiaoxiangmoe** [comment]
@jtsom If you use vscode-ng-language-service v0.1100.1, please reinstall v0.1100.0. It seems that vscode-ng-language-service v0.1100.1 has *serious bugs* and ivy language service will not work.

**Comment by jtsom** [comment]
Tried that. v0.1100.0 does the same thing - everything just says "typescript"

**Comment by xiaoxiangmoe** [comment]
run `yarn ngcc` and then reopen vscode

**Comment by jtsom** [comment]
I don't have yarn installed and don't plan on installing it for a workaround to a bug.

I also just saw 0.1100.2 was available and installed and tried that. It looks like whatever was broken was fixed in this version. I will do some more testing to make sure.

![image](https://user-images.githubusercontent.com/13488/101161780-57266a00-35ff-11eb-9c87-65b282411727.png)

**Comment by xiaoxiangmoe** [comment]
I tried 0.1100.2 and it doesn't work.

This is my `.vscode/settings.json`

```json
{
  "angular.experimental-ivy": true
}
```

This is my angularCompilerOptions
```json
{
  "angularCompilerOptions": {
    "strictInjectionParameters": true,
    "strictInputAccessModifiers": true,
    "strictTemplates": true
  }
}
```

**Comment by lincolnthree** [comment]
Same problem here with "angular.experimental-ivy:true". Also tried latest manual install (0.1101.0-next.1)

An issue appears to be the concatenation of paths in the vscode extension when locating the `tsconfig.json` file:

```
[Info  - 11:28:47 AM] Angular language server process ID: 33010
[Info  - 11:28:47 AM] Using typescript/lib/tsserverlibrary v4.0.5 from /Users/lincoln/projects/maritlabs/topdecked-unified/node_modules/typescript/lib/tsserverlibrary.js
[Info  - 11:28:47 AM] Using @angular/language-service/bundles/ivy v11.1.0-next.3 from /Users/lincoln/.vscode/extensions/angular.ng-template-0.1101.0-next.1/node_modules/@angular/language-service/bundles/ivy.js
[Info  - 11:28:47 AM] Log file: /Users/lincoln/Library/Application Support/Code/logs/20201217T103253/exthost1/Angular.ng-template/nglangsvc.log
[Error - 11:28:53 AM] Cannot read file '/Users/lincoln/projects/maritlabs/topdecked-unified/Users/lincoln/projects/maritlabs/topdecked-unified/tsconfig.base.json'.
[Info  - 11:28:54 AM] Enabling Ivy language service for /Users/lincoln/projects/maritlabs/topdecked-unified/tsconfig.json.
```

Note this line:

```
Cannot read file '/Users/lincoln/projects/maritlabs/topdecked-unified/Users/lincoln/projects/maritlabs/topdecked-unified/tsconfig.base.json'.
```

This should clearly be `/Users/lincoln/projects/maritlabs/topdecked-unified/tsconfig.base.json`. Not sure why this is happening. I don't think I have any special configuration for file paths.

Happy to provide the nglangsvc.log file (directly to devs) as well but cannot post online for security reasons.

**Comment by lincolnthree** [comment]
This now appears to be working better as of the latest pre-release `v0.1101.0-next.2
`. Our project is detected and IVY language service appears to be activated, but possibly a bit slow and some features appear to be missing compared to View LS. Filing issue reports as I notice gaps.

**Comment by kyliau** [comment]
The new Ivy-native language service has been released in [v11.1.0](https://github.com/angular/vscode-ng-language-service/releases/tag/v11.1.0).
It's an opt-in feature for now, please give it a try and let us know if you have any feedback.
For the best editor experience, please make sure your project has [`strictTemplates`](https://angular.io/guide/angular-compiler-options#stricttemplates) enabled in `angularCompilerOptions`.
